.cspell.json

```diff
@@ -47,7 +47,6 @@
   // words - list of words to be always considered correct
   "words": [
     "RUSTPYTHONPATH",
-    // RustPython
```

Comment:
This is meaning the terms below are not coming from python or cpython, but rustpython-original terms